Kickstarter-iOS/ViewModels/AppDelegateViewModel.swift

```diff
@@ -1087,3 +1094,31 @@ private func qualtricsConfigData() -> QualtricsConfigData {
     .compact()
   )
 }
+
+private func shouldSeeCategoryPersonalization() -> Bool {
+  let isLoggedIn = AppEnvironment.current.currentUser != nil
+  let hasSeenCategoryPersonalization = AppEnvironment.current.userDefaults.hasSeenCategoryPersonalizationFlow
+
+  if isLoggedIn || hasSeenCategoryPersonalization {
+    // Currently logged-in users should not see the onboarding flow
+    AppEnvironment.current.userDefaults.hasSeenCategoryPersonalizationFlow = true
+
+    return false
+  }
+
+  let variant = AppEnvironment.current.optimizelyClient?.variant(
```

Comment:
So this will still load async and we may briefly see the discover feed?

Reply:
That's right!